### ISSUE TITLE
Pyros issues

### ISSUE THREAD

**Issue by XargonWan** [omf2097/openomf#367]
Sometimes Pyros grabs "the nothing"meaning that the sprite of the opponent is not put on Pyro's "hand".

With Pyros <- + PUNCH I can easly overkill a cornered opponent.

I believe I can fly: sometimes I can throw my opponent up in the sky, but I don't know how I did it, I was using Pyros.

**Comment by katajakasa** [comment]
Yup, this is due to unknown physics and animation tag behaviour. Pyros  and Gargoyle are currently our greatest offenders in this regard.

Known bug. Marking as such.

**Comment by LegendaryBlueShirt** [comment]
I want to say this one's done, not aware of any pyros moves that have issues anymore.
